config/locales/client.en.yml

```
@@ -33,14 +33,20 @@ en:
       other: answers
     qa:
       post:
-        submit_comment: "comment"
-        edit_comment: "edit"
-        cancel_comment: "cancel"
-        add_comment: "Add a comment"
-        show_comment:
-          one: Show %{count} more comment
-          other: Show %{count} more comments
-        delete_comment_confirm: "Are you sure you want to delete this comment?"
+        qa_comment:
+          composer:
+            too_short: "Enter at least %{count} characters"
+            length_ok: "%{count} characters left..."
+            too_long: "Comment cannot be longer than %{count} characters"
```

Needs pluralization.

```suggestion
            too_short:
              one: "Enter at least %{count} character"
              other: "Enter at least %{count} characters"
            length_ok:
              one: "%{count} character left..."
              other: "%{count} characters left..."
            too_long:
              one: "Comment cannot be longer than %{count} character"
              other: "Comment cannot be longer than %{count} characters"
```

---

BTW: I just noticed that this file contains a couple of `{{count}}` interpolation keys. That format is deprecated and linting should have warned you about it. Not sure why it didn't. :thinking: Anyway, it should be `%{count}`

---

Good catch :+1: I've fixed these up.